common/persistence/nosql/nosqlplugin/cassandra/workflowUtils.go

```diff
@@ -1205,8 +1203,13 @@ func (db *cdb) updateWorkflowExecution(
 ) error {
 	execution.StartTimestamp = db.convertToCassandraTimestamp(execution.StartTimestamp)
 	execution.LastUpdatedTimestamp = db.convertToCassandraTimestamp(execution.LastUpdatedTimestamp)
-
+	//default TTL Value. 0 TTL means no ttl is set, hence your records will persist forever unless explicitly deleted.
+	ttlInSeconds := 0
+	if db.dc.EnableExecutionTTL(domainID) && execution.TTLInSeconds >= 0 {
```

Comment:
If we add another nosql plugin, we need this logic as well. Can you move the feature flag to where you calculate it?

Reply:
as shijie said, could you move the feature flag to that place? That we don't need to do check in all types of databases.